addons/l10n_co/models/res_partner.py

```diff
@@ -0,0 +1,24 @@
+# coding: utf-8
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.depends('vat')
+    def _compute_verification_code(self):
```

Comment:
You are correct: https://www.odoo.com/documentation/10.0/reference/guidelines.html#symbols-and-conventions.